README.md

```diff
@@ -8,7 +8,7 @@ Two methods of installation are provided: A "dev" deployment on a virtual machin
 
 You can either use [Vagrant](https://www.vagrantup.com/) to create a complete virtual machine for you, or set up a VM manually.
 
-Either way, you will need to edit some parts of `group_vars/dev/vars.yml` for your setup.
+Either way, you will need to edit some parts of `group-vars/dev/vars.yml` for your setup.
```

Comment:
It's definitely an underscore not a dash on my machine!

Reply:
fin but it needs to be the same as itis above

Reply:
That's weird. I didn't touch those lines